### PR TITLE
completed javascript modal

### PR DESCRIPTION
**Pull request by hnguy96**
Screen Shot 2021-03-23 at 3 05 35 PM
Screen Shot 2021-03-23 at 3 06 05 PM
Screen Shot 2021-03-23 at 3 06 18 PM
Screen Shot 2021-03-23 at 3 06 24 PM

### CODE REVIEW

File: javascript-modal/styles.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+.modal > button {
+  padding: 0.5rem 2.1rem;
+  background-color: rgb(255, 81, 0);
+  text-transform: uppercase;
+}
+
+.show {
+  visibility: visible;
+}
```
Comment (codingcodymiller) [inline javascript-modal/styles.css:68]: This class can be removed

File: javascript-modal/styles.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+.hidden {
+  visibility: hidden;
+}
```
Comment (codingcodymiller) [inline javascript-modal/styles.css:72]: I would make this `display: none` instead. `visibility` still leaves its elements with size on the page, which is typically not desirable.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work so far!! A couple minor notes:

**Comment by hnguy96** [comment]
Screen Shot 2021-03-23 at 3 42 01 PM

File: javascript-modal/main.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+var $openModalBtn = document.querySelector('.open-modal-btn');
+var $modalContainer = document.querySelector('.modal-container');
+var $noBtn = document.querySelector('.no-btn');
+
+function showModal(event) {
+  $modalContainer.classList.replace('hidden', 'show');
+}
+
+function hideModal(event) {
+  $modalContainer.classList.replace('show', 'hidden');
```
Comment (codingcodymiller) [inline javascript-modal/main.js:10]: Now that you don't have a `show` class anymore, you should revise this.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-modal/main.js (hunk 3 above)
Comment (hnguy96) [reply javascript-modal/main.js:10]: doesn't `classList.replace` replace class "hidden" with class "show"?

**Comment by hnguy96** [comment]
Screen Shot 2021-03-23 at 4 18 32 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**